.dopplerci

```diff
@@ -0,0 +1 @@
+Jenkinsfile
```

Comment:
The name is `.doppler-ci`, and ensure that it is a symlink to a valid Jenkinsfile:

> In Windows (with git bash as administrator): `export MSYS=winsymlinks:nativestrict; ln -s Jenkinsfile .doppler-ci`

See https://github.com/FromDoppler/hello-microservice#prepare-the-files